packages/web3js-plugin/src/plugin.ts

```diff
@@ -36,7 +36,7 @@ export class OptimismPlugin extends Web3PluginBase {
    */
   public async getBaseFee<
     ReturnFormat extends DataFormat = typeof DEFAULT_RETURN_FORMAT
-  >(returnFormat?: ReturnFormat) {
+  >(returnFormat?: ReturnFormat): Promise<bigint> {
```

Comment:
The return type of these functions is a bit complex since it's defined by the user via `returnFormat`. It was deemed easier to just let TypeScript infer the type using `returnFormat` and web3.js' [FormatType](https://github.com/web3/web3.js/blob/2b445bf2c02e66adff4e428907dd39347f6085c9/packages/web3-types/src/data_format_types.ts#L52). I would suggest we revert these return type changes

Reply:
The reason I had to make these changes was because the build errors. Feel free to push to this branch though - i realize I am not a typescript expert :)

Reply:
Seperate comment, I usually close all dependabot prs that aren't trivial to merge because otherwise it's easy for dependabot to be creating more work than it's saving

Reply:
Totally, please close this pr then if you're gonna handle separately :)